Reconstruction/RecCalorimeter/src/components/CreateCaloCells.h

```diff
@@ -56,25 +58,28 @@ class CreateCaloCells : public GaudiAlgorithm {
   ToolHandle<ICalorimeterTool> m_geoTool;
 
   /// Calibrate to EM scale?
-  bool m_doCellCalibration;
+  Gaudi::Property<bool> m_doCellCalibration{this, "doCellCalibration", true, "Calibrate to EM scale?"};
   /// Add noise to cells?
-  bool m_addCellNoise;
+  Gaudi::Property<bool> m_addCellNoise{this, "addCellNoise", true, "Add noise to cells?"};
   /// Save only cells with energy above threshold?
-  bool m_filterCellNoise;
+  Gaudi::Property<bool> m_filterCellNoise{this, "filterCellNoise", false, "Save only cells with energy above threshold?"};
   /// Handle for calo hits (input collection)
   DataHandle<fcc::CaloHitCollection> m_hits;
   /// Handle for calo cells (output collection)
   DataHandle<fcc::CaloHitCollection> m_cells;
   /// Name of the detector readout
-  std::string m_readoutName;
-  /// Name of active volumes
-  std::string m_activeVolumeName;
+  Gaudi::Property<std::string> m_readoutName{this, "readoutName", "ECalHitsPhiEta", "Name of the detector readout"};
+  /// Name of active volumes (material name)
+  Gaudi::Property<std::string> m_activeVolumeName{this, "activeVolumeName", "_sensitive", "Name of the active volumes (material name)"};
   /// Name of active layers for sampling calorimeter
-  std::string m_activeFieldName;
-  /// Name of the fields describing the segmented volume
-  std::vector<std::string> m_fieldNames;
-  /// Values of the fields describing the segmented volume
-  std::vector<int> m_fieldValues;
+  Gaudi::Property<std::string> m_activeFieldName{this, "activeFieldName", "active_layer", "Name of active layers for sampling calorimeter"};
+  /// Name of the bit-fields (in the readout) that contain the volume to segment
```

Comment:
segmented -> to segment changes logic (it is already segmented).
Maybe "Name of the bit-fields (in the readout) describing the volume"? Same below for values